front/src/modules/ui/components/property-box/PropertyBox.tsx

```diff
@@ -7,7 +7,7 @@ const StyledPropertyBoxContainer = styled.div`
   border-radius: ${({ theme }) => theme.border.radius.sm};
   display: flex;
   flex-direction: column;
-  gap: ${({ theme }) => theme.spacing(2)};
+  gap: ${({ theme }) => theme.spacing(0.5)};
```

Comment:
I'm approving this PR but we need to have a discussion with Thomas on spacing between. The goal of the spacing function is to enforce spacing to be a multiple of 4px. Here we are hacking it to get 2px.
So should the step be 2px? Is this an exception for spacing between vertical elements?